src/main/java/seedu/duke/DeleteCommand.java

```diff
@@ -0,0 +1,20 @@
+package seedu.duke;
+
+public class DeleteCommand implements Command {
+    private final InternshipList internships;
+    private int index;
+
+    public DeleteCommand(InternshipList internshipList, int index) {
+        this.internships = internshipList;
+        this.index = index;
+    }
+    public void execute (String[] args) {
+        index = Integer.parseInt(args[1]);
+        int id = index - 1;
```

Comment:
Would it be better if we interchanged the names of the 'index' and 'id' variables here ? because the user will be referring to the id of the internship in their argument right ? And while deleting the internship, we'll refer to the actual index of the internship in the array which is (id - 1). Let me know if my understanding is correct because I just went through all the code you committed bro haha

Reply:
You are right rahul. Have amended accordingly! Thanks for spotting the mistake!